src/ngraph/runtime/cpu/builder/dot.cpp

```diff
@@ -250,7 +250,8 @@ namespace ngraph
                            arg0_shape,
                            arg1_shape,
                            result_shape,
-                           reduction_axes_count);
+                           reduction_axes_count,
+                           1.0f);
```

Comment:
What is this magic number?

Reply:
Requantization scale which is 1 for non quant dots.

Reply:
A comment that denotes 1.0 as a requant scale would be good.